aws/index.ts

```diff
@@ -37,28 +40,27 @@ const namespace = new k8s.core.v1.Namespace(
   { provider: cluster.k8sProvider },
 );
 
-// Operator namespace name
-export const operatorNamespace = namespace.metadata.name;
+if (config.installAkkaOperator) {
+  const serviceAccountName = util.name("sa");
```

Comment:
What is `"sa"`? Should that be a parameter or something configurable? It looks like a magic number/value. Maybe add comment about why it has to be `"sa"`. WDYT?

Reply:
This is a name of a service account. It can be anything, not necessarily `"sa"`. We can introduce a config setting but I'm not sure it's needed.